crates/archive/Cargo.toml

```diff
@@ -19,3 +19,4 @@ cfg-if = "1.0"
 headers = "0.4"
 thiserror = "1.0.16"
 attohttpc = { version = "0.28", default-features = false, features = ["json", "compress", "tls-rustls-native-roots"] }
+log = { version = "0.4", features = ["std"] }
```

Comment:
I included this here originally as part of the debugging work in #1784, but it seems reasonable to just have it here regardless. It would also be good to update these kinds of dependencies to be `log.workspace = true` instead of specifying their own dependencies, but that’s for a different change!

Reply:
Seems reasonable! I agree it's worth updating our use of Cargo workspaces—I think this repo pre-dates a lot of the improvements in that space that Cargo has shipped—but also agree it's for a separate PR and not a blocker for anything.